test/node-api/test_make_callback/binding.c

```diff
@@ -44,13 +46,40 @@ static napi_value MakeCallback(napi_env env, napi_callback_info info) {
   return result;
 }
 
+static void AsyncDestroyCb(napi_env env, void* data, void* hint) {
+  napi_status status = napi_async_destroy(env, (napi_async_context)data);
+  assert(status == napi_ok);
```

Comment:
You may want to add a comment that `NAPI_ASSERT_RETURN_VOID()` is not usable here.

Reply:
Yup, done!